cmd/run.go

```diff
@@ -312,7 +319,10 @@ dapr run --app-id myapp --app-port 3000 --app-protocol grpc -- go run main.go
 			}
 		}
 
-		if output.AppCMD != nil && (output.AppCMD.ProcessState == nil || !output.AppCMD.ProcessState.Exited()) {
+		if output.AppErr != nil {
+			exitWithError = true
+			print.FailureStatusEvent(os.Stderr, fmt.Sprintf("Error exiting App: %s", output.AppErr))
+		} else if output.AppCMD != nil && (output.AppCMD.ProcessState == nil || !output.AppCMD.ProcessState.Exited()) {
 			err = output.AppCMD.Process.Kill()
 			if err != nil {
 				print.FailureStatusEvent(os.Stderr, fmt.Sprintf("Error exiting App: %s", err))
```

Comment:
@shubham1172  If it errors out here and line number 316 then `exitWithError`  will not be set.  Is this expected ? or it is not possible to have this behavior. Just wanted to bring it your notice.

Reply:
Yes @pravinpushkar, that was an existing behavior that I haven't changed with this PR. Now that I think more about it I think we should exit with error in that scenario too. What do you think? 
cc @mukundansundar

Reply:
Have added the same to this PR!